service/src/main/kotlin/no/nav/su/se/bakover/service/SendPåminnelseNyStønadsperiodeServiceImpl.kt

```diff
@@ -0,0 +1,151 @@
+package no.nav.su.se.bakover.service
+
+import arrow.core.Either
+import arrow.core.flatMap
+import arrow.core.right
+import no.nav.su.se.bakover.common.periode.Periode
+import no.nav.su.se.bakover.common.persistence.SessionFactory
+import no.nav.su.se.bakover.domain.JobContextRepo
+import no.nav.su.se.bakover.domain.Sak
+import no.nav.su.se.bakover.domain.Saksnummer
+import no.nav.su.se.bakover.domain.SendPåminnelseNyStønadsperiodeContext
+import no.nav.su.se.bakover.domain.brev.LagBrevRequest
+import no.nav.su.se.bakover.domain.dokument.Dokument
+import no.nav.su.se.bakover.domain.sak.SakRepo
+import no.nav.su.se.bakover.service.brev.BrevService
+import no.nav.su.se.bakover.service.person.PersonService
+import org.slf4j.LoggerFactory
+import java.time.Clock
+import java.time.LocalDate
+
+interface SendPåminnelseNyStønadsperiodeService {
+    fun sendPåminnelser(): Either<KunneIkkeSendePåminnelse, SendPåminnelseNyStønadsperiodeContext>
```

Comment:
Tenker denne bare kan være void dersom den bare skal brukes async.

Reply:
Lar det bare være, greit for å kunne asserte i tester etc.